.build/cspell-words.txt

```diff
@@ -10,6 +10,7 @@ bincirc
 BSTR
 cach
 cachtokn
+Calendrier
```

Comment:
I would like to see if there is a way to avoid this.

Reply:
Agree.  Will remove until we get a complete solution.